publish.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash -ex
 
-docker pull registry.tld/conjurinc/publish-rubygem
+echo "*** Starting Publish"
 
+git clone git@github.com:conjurinc/release-tools.git
```

Comment:
Not sure if this should be the Atlantis repo now?

Reply:
No can do from github.com